cluster/manifests/skipper-canary-controller/canary-cronjob.yaml

```diff
@@ -30,6 +30,10 @@ spec:
           - name: skipper-canary-controller
             terminationMessagePolicy: FallbackToLogsOnError
             image: container-registry.zalando.net/gwproxy/skipper-canary-controller:main-29
+            lifecycle:
+              preStop:
+                sleep:
+                  seconds: 30
```

Comment:
If we need any delay it should added into the application shutdown hook.
Also value of 30 equals default termination grace period so this leaves no time for pod to shutdown.